trigger/version.go

```diff
@@ -293,14 +293,18 @@ func (t *versionTriggers) versionRegression(sub *event.Subscription) (*notificat
 	return nil, nil
 }
 
-func (t *versionTriggers) waitOnChildrenOrSiblings(sub *event.Subscription) (bool, error) {
+func (t *versionTriggers) waitOnChildrenOrSiblings() (bool, error) {
+	// only patches have to wait on children/siblings
+	if !evergreen.IsPatchRequester(t.version.Requester) {
+		return true, nil
+	}
 	isReady := false
 	patchDoc, err := patch.FindOne(patch.ByVersion(t.version.Id))
 	if err != nil {
-		return isReady, errors.Wrapf(err, "error getting patchDoc '%s'", t.version.Id)
+		return isReady, errors.Wrapf(err, "error getting patch '%s'", t.version.Id)
```

Comment:
sorry for the unrelated change; it's best for errors to say what the error actually is rather than using variable names